apps/docs/src/components/Layout.jsx

```diff
@@ -38,11 +50,7 @@ export const navigation = [
   },
   {
     title: 'Contributing',
-    links: [
-      { title: 'How to contribute', href: '/docs/how-to-contribute' },
```

Comment:
I'd rather leave the TODO page that we can write later or remove apps/docs/src/app/docs/how-to-contribute/page.md